src/relion/cryolo_relion_it/icebreaker_histogram.py

```diff
@@ -0,0 +1,29 @@
+import gemmi
+import json
+import matplotlib.pyplot as plt
+
+
+def extract_ice_column(icebreaker_star_file_path):
+    particles = gemmi.cif.read_file(icebreaker_star_file_path)
+    data_as_dict = json.loads(particles.as_json())["particles"]
+    print(data_as_dict)
+    ice_group = data_as_dict["_rlnhelicaltubeid"]
+    print(ice_group)
+    return ice_group
+
+
+def create_histogram(working_directory):
+    icebreaker_path = str(working_directory / "External/Icebreaker_group/")
+    try:
+        data = extract_ice_column(icebreaker_path + "/particles.star")
+        plt.hist(x=data, bins="auto", rwidth=0.9)
+        plt.xlabel("Ice thickness")
+        plt.ylabel("Number of particles")
+        plt.title("Estimated ice thickness")
```

Comment:
Just wondering, do you know the units for this?

Reply:
(estimated ice thickness)

Reply:
I'm not sure, I seem to remember from Mat's talk that it isn't an absolute thickness, but a relative one. I asked and he also isn't sure what the units are. Apparently he and Yuriy are planning to take some real measurements and see how the Icebreaker estimated values match up with the real thickness.